### DISCUSSION TITLE
Are wildcard urls("https://*.our-deployment.com") supported in developer dashboard?

### DISCUSSION THREAD

**Discussion by YZhenY** [Web3Auth/Web3Auth#306] [Help]
As listed

**Comment by BboyStatix** [comment]
No, not supported. The redirection endpoint URI MUST be an absolute URI. 
As defined by [RFC3986] Section 4.3. 
* The endpoint URI MAY include an "application/x-www-form-urlencoded" formatted (per Appendix B) query component ([RFC3986] Section 3.4), which MUST be retained when adding additional query parameters.
* The endpoint URI MUST NOT include a fragment component."

**Comment by chaitanyapotti** [comment]
You can use the wildcard urls w/o using the dashboard.
https://github.com/orgs/Web3Auth/discussions/351
